From 475981617585d691be0c8599885aaf5953c3dfa2 Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@gmail.com>
Date: Wed, 25 Dec 2024 12:39:51 -0500
Subject: [PATCH] Turn on the power led and turn off status led on sunxi-h6

Signed-off-by: Patrick Yavitz <pyavitz@gmail.com>
---
 board/sunxi/board.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/board/sunxi/board.c b/board/sunxi/board.c
index 961cdcde74..7eaeab24a9 100644
--- a/board/sunxi/board.c
+++ b/board/sunxi/board.c
@@ -550,6 +550,11 @@ void sunxi_board_init(void)
 {
 	int power_failed = 0;
 
+#if defined(CONFIG_MACH_SUN50I_H6)
+	gpio_direction_output(SUNXI_GPL(4), 0);
+	gpio_direction_output(SUNXI_GPL(7), 1);
+#endif
+
 #ifdef CONFIG_LED_STATUS
 	if (IS_ENABLED(CONFIG_SPL_DRIVERS_MISC))
 		status_led_init();
-- 
2.39.5

